pycbc/__init__.py

```diff
@@ -45,7 +46,21 @@
 __version__ = pycbc_version
 
 
-def init_logging(verbose=False, format='%(asctime)s %(message)s'):
+_default_format = '%(asctime)s %(message)s'
+
+
+class LogFormatter(logging.Formatter):
```

Comment:
Could you please add a docstring to describe why we use this custom class, indicating the example timestamp format we discussed?